### PR TITLE
Fix missing include

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

File: include/gridtools/stencil/frontend/cartesian/dimension.hpp [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 #include "../../../common/defs.hpp"
 #include "../../../common/host_device.hpp"
+#include <cstddef>
```
Comment (fthaler) [inline include/gridtools/stencil/frontend/cartesian/dimension.hpp:14]:
```suggestion
#include <cstddef>

#include "../../../common/defs.hpp"
#include "../../../common/host_device.hpp"
```

Comment (fthaler) [reply include/gridtools/stencil/frontend/cartesian/dimension.hpp:14]: This is the include order we use in most other files as far as I can tell.

**Comment by havogt** [comment]
launch jenkins

**Pull request merged**